docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to Endless Sky
+
+## Posting issues
+
+The [issues page](https://github.com/endless-sky/endless-sky/issues) on GitHub is for tracking bugs and feature requests. When posting a new issue, please:
+
+* Check to make sure it's not a duplicate of an existing issue.
+* Create a separate "issue" for each bug you are reporting and each feature you are requesting.
+* Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
+
+If requesting a new feature, first ask yourself: will this make the game more fun or interesting? Remember that this is a game, not a simulator. Changes will not be made purely for the sake of realism, especially if they introduce needless complexity or aggravation.
+
+## Posting pull requests
+
+If you are posting a pull request, please:
+
+* Do not combine multiple unrelated changes.
+* Check the diff and make sure the pull request does not contain unintended changes.
+* If changing the C++ code, follow the [coding standard](https://endless-sky.github.io/styleguide/styleguide.xml).
+
+If proposing a major pull request, start by posting an issue and discussing the best way to implement it. Often the first strategy that occurs to you will not be the cleanest or most effective way to implement a new feature.
+
+## Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your PR gets closed because a PR was merged, and you are not satisfied, please open a new issue.
```

Comment:
```suggestion
If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
```

Reply:
oops